src/components/button/button.tsx

```diff
@@ -201,6 +201,18 @@ export class Button
 
   componentDidLoad(): void {
     setComponentLoaded(this);
+
+    const slotted = this.el.shadowRoot
+      .querySelector("span.content")
+      .querySelector("slot") as HTMLSlotElement;
+
+    const textNodes = slotted.assignedNodes().filter((node) => node.nodeName === "#text");
```

Comment:
Do you need the textNodes or can you just see if the parent node will work?

Reply:
Could you expand on this one?